feed/templates/index.html

```diff
@@ -33,6 +34,12 @@ <h2 class="card-title">{{ post.title }}</h2>
             {% endfor %}
         </div>
         {% block sidebar %} {% include 'sidebar.html' %} {% endblock sidebar %}
+    <!-- advert-->
+    {% if advert %}
+    {{ image }}
+    {{ advert_url}}
```

Comment:
```suggestion
    {{ advert.url}}
    {{ advert.image.url }}
```

This also belongs in your post_detail.html template. The reason is because you added the `context` to your PostDetail() view, which is rendering "post_detail.html". Cut and paste this to the other template.